### PR TITLE
Feature/greeting

### PR DESCRIPTION
**Pull request by AmineDerbal**
in this pull request ♻️ 

- Create the react front end app for our [back end](https://github.com/AmineDerbal/Hello-back-end-app) 👍 
- Install and run the back end first before running the front end 👍

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,6 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Greeting />;
```
Comment (AmaduKamara) [inline src/App.js]: - [ ] For this file, it's expected that a route is present that routes to the Greeting component and is done with the help of react-router-dom. Kindly set up react-router and add a link that routes to the Greeting component from the App.js 👍🏽

Screenshot 2023-07-11 at 10 23 28 AM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @AmineDerbal,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Reads random greetings from the API 👍🏽
✅ Used redux correctly 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly set up react-router on your project so there is a route from the App.js component that links to the Greeting.js page. It's required in the project requirements that react-router is used for routing. Please add that and all will be fine 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 1 above)
Comment (AmineDerbal) [reply src/App.js]: @AmaduKamara 
Understood I will fix this 👍 
Thanks for your feeback 😄

**Comment by AmineDerbal** [comment]
Add Routing to the app 👍 
Setup the Greeting component as a route in the App.js file 👍

**Review by mckent05 — APPROVED** [review]
Hi @AmineDerbal, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job adding a route to your app 👍 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**